ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -1626,6 +1626,23 @@ periodics:
         value: /usr/local/osde2e-common,/usr/local/crw-osd-token
       - name: CONFIGS
         value: aws,stage,addon-suite
+      - name: OCM_CCS
+        value: "true"
+      - name: OCM_AWS_ACCOUNT
+        valueFrom:
+          secretKeyRef:
+            key: ocm-aws-account
+            name: ocm-aws-access
+      - name: OCM_AWS_ACCESS_KEY
+        valueFrom:
+          secretKeyRef:
+            key: ocm-aws-account
```

Comment:
Is this really the correct secret key? It's the same you are using for `OCM_AWS_ACCOUNT`

Reply:
Yes, I'm using the same secret with three key-value pairs

Reply:
sorry, my mistake, fixed